### ISSUE TITLE
Error: module org.bytedeco.opencv does not export org.bytedeco.opencv.helper

### ISSUE THREAD

**Issue by AKMANSoft** [bytedeco/javacpp-presets#927]
We have old app, which we update to JDK 11, but I cannot access the cvContoure function, which I check use the helper package, also I have check helper package is not in module.info file of JavaCV version of OpenCV.

**Comment by saudet** [comment]
Good catch! If possible, please send a PR for that one and any other presets.

/cc @HGuillemet

**Comment by AKMANSoft** [comment]
Sorry I don't understand "please send a PR for that one and any other presets" PR??

**Comment by saudet** [comment]
A pull request, like this: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request
